trunner/test_runner.py

```diff
@@ -104,9 +104,14 @@ def search_for_tests(self) -> List[Path]:
             yamls = []
 
             if path.is_dir():
-                yamls = list(path.rglob("test*.yaml")) + list(path.rglob("test*.yml"))
+                for pattern in ["test*.yaml", "test*.yml"]:
+                    for p in path.rglob(pattern):
+                        if ".git" not in str(p):
```

Comment:
Maybe sth like this (not tested):

```suggestion
                for path in list(path.rglob("test*.yaml")) + list(path.rglob("test*.yml")):
                        if ".git" not in str(p):
```

Reply:
Could be done in one line, but I don't know if it would not lose readability
`                yamls = [ p for p in list(path.rglob("test*.yaml")) + list(path.rglob("test*.yml")) if ".git" not in str(p) ]`

Reply:
I'd leave the current solution for readability reasons (as you said).